### ISSUE TITLE
Panic with skaffold dev in 1.13.2

### ISSUE THREAD

**Issue by milas** [GoogleContainerTools/skaffold#4748]
This is happening in 1.13.2 as well as latest build (b642a69 as of issue time). It does NOT happen in 1.13.1.

Given the panic, it appears to have been introduced by #4676.

### Expected behavior
* Logs start streaming and containers run normally

### Actual behavior
* Logs start streaming
* After a moment or two, skaffold panics
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x17b3bea]

goroutine 394 [running]:
github.com/GoogleContainerTools/skaffold/pkg/skaffold/trigger.(*pollTrigger).Start.func1(0xc000706230, 0xc000469be0, 0xc0006c4de0, 0x215bea0, 0xc0005ad380)
        /skaffold/pkg/skaffold/trigger/triggers.go:105 +0xda
created by github.com/GoogleContainerTools/skaffold/pkg/skaffold/trigger.(*pollTrigger).Start
        /skaffold/pkg/skaffold/trigger/triggers.go:99 +0x9f
```

### Information
- Skaffold version: 1.13.2 (also happens on b642a69)
- Local Cluster: k3d
- Operating system: Manjaro Linux
- Contents of skaffold.yaml: can't paste here, but at a high-level has some Jib builds, has some local Docker builds, deployments are using K8S manifests, and I'm launching using a profile

### Steps to reproduce the behavior

1. `skaffold dev -p <profile> --port-forward`

**Comment by briandealwis** [comment]
@gsquared94 can you take a look?  It looks like this creation of `pollTrigger` is missing initialization of `isActive`:
https://github.com/GoogleContainerTools/skaffold/blob/b91bfbe794d5cf01a8c4e8a3e6f8b445ce7702ba/pkg/skaffold/trigger/triggers.go#L268-L270

**Comment by gsquared94** [comment]
> @gsquared94 can you take a look? It looks like this creation of `pollTrigger` is missing initialization of `isActive`:
> 
> https://github.com/GoogleContainerTools/skaffold/blob/b91bfbe794d5cf01a8c4e8a3e6f8b445ce7702ba/pkg/skaffold/trigger/triggers.go#L268-L270

@briandealwis you're right. This is the issue. Fixed it in #4750.

**Comment by milas** [comment]
I just built #4750 locally and can confirm I'm not getting a panic when starting up anymore 🎉
